src/components/Login/Login.jsx

```diff
@@ -0,0 +1,94 @@
+import { useState } from 'react';
+import axios from 'axios';
+import { Link as RouterLink, useNavigate } from 'react-router-dom';
+import { BACKEND_LOGIN_URL } from '../../constants/constants';
+import { LockOutlined, UserOutlined } from '@ant-design/icons';
+import { Button, Form, Input, Typography } from 'antd';
+import { message } from 'antd';
+import storageService from '../../services/storageService';
+import './Login.css';
+import { LOGIN_ALERTS } from '../../constants/constants';
+
+const { Title } = Typography;
+
+const Login = () => {
+  const navigate = useNavigate();
+  const [name, setName] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleSubmit = async (e) => {
+    try {
+      const response = await axios.post(BACKEND_LOGIN_URL, { name, password });
+      const token = response.data;
```

Comment:
Don`t we have a function in the `userServie` of logic? if not please put this into a function inside the service